index.html

```diff
@@ -112,6 +112,26 @@ <h3>
           Clients can use the presence or absence of this method to determine
           whether to show UI that triggers its use.
         </div>
+        <section>
+          <h4>
+            Internal Slots
+          </h4>
+          <p>
+            This API adds the following internal slot to the {{Navigator}}
+            interface.
+          </p>
+          <dl>
+            <dt>
+              {{Promise}}? <dfn data-dfn-for=
+              "Navigator">[[\sharePromise]]</dfn>
+            </dt>
+            <dd>
+              The <a>[[\sharePromise]]</a> is a promise that represents a
+              user's current intent to share some data with a <a>share
+              target</a>. It is initialized to `null`.
```

Comment:
Backticks won't have any effect, will they?

In my specs (at least WST), I just leave the word "null" unformatted. How do you format it?

Reply:
ReSpec now formats it with `<code>null</code>`: 
![Screenshot 2019-08-16 14 25 21](https://user-images.githubusercontent.com/870154/63143455-b559c300-c031-11e9-9fc3-a9545639233b.png)